### PR TITLE
add jquery

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by dougalsutherland** [comment]
See discussion at https://github.com/conda-forge/staged-recipes/pull/4044#issuecomment-332019807.

**Comment by sannykr** [comment]
@dougalsutherland : Could you please look at the CI log. Now the tar file is not empty.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Our discussion about non python packages was resolved. Could you please review and merge. It fails only for python35 on window due to some system issues.

File: recipes/jquery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    
+test:
+  imports:
+
```
Comment (isuruf) [inline recipes/jquery/meta.yaml]: Please add a test. Maybe check that a file exists

File: recipes/jquery/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  noarch: True
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
```
Comment (isuruf) [inline recipes/jquery/meta.yaml]: Remove this

File: recipes/jquery/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jquery/jquery/archive/3.2.1.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: True
```
Comment (isuruf) [inline recipes/jquery/meta.yaml]: `noarch: generic`

File: recipes/jquery/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{% set name = "jquery_static" %}
+{% set version = "3.2.1" %}
+{% set sha256 = "31d8c2e22dd48a7973fe3617cc656fe6b64d2250d07a258cd8ab30ead031c24a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jquery/jquery/archive/3.2.1.tar.gz
```
Comment (isuruf) [inline recipes/jquery/meta.yaml]: Use jinja template for the version

File: recipes/jquery/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    
+test:
+  commands:
+    - npm list -g jquery
```
Comment (dougalsutherland) [inline recipes/jquery/meta.yaml]: You'll need `nodejs` in `test/requires` to run this, since it isn't a run requirement.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Please let me know if you have more suggestions.

**Comment by scopatz** [comment]
@isuruf - does anything else need to happen here?

**Comment by isuruf** [comment]
Thanks @sannykr

**Pull request merged**